doc/source/whatsnew/v1.5.0.rst

```diff
@@ -1034,6 +1034,10 @@ Metadata
 - Fixed metadata propagation in :meth:`DataFrame.explode` (:issue:`28283`)
 -
 
+Visualization
```

Comment:
1. Sorry, there's an existing `Plotting` above where this note can go instead.
2. The note should be describe in terms of the public API, e.g. `DataFrame.plot`, could you note which public method was used to hit this bug?